openupgradelib/openupgrade_tools.py

```diff
@@ -368,9 +368,11 @@ def invalidate_cache(env, flush=True):
     # Warning on possibly untested versions where chunked might not work as expected
     if version > 17:  # unreleased version at this time
         logger.warning(
-            f"openupgradelib.invalidate_cache hasn't been tested on Odoo {version}. "
-            "Please report any issue you may find and consider bumping this warning "
-            "to the next version otherwise."
+            """openupgradelib.invalidate_cache hasn't been tested on Odoo {}.
```

Comment:
Don't use triple quote, as that preserves the line feeds and indentations.

Reply:
What should I use?

Reply:
Simple quotes, as they were before.

Reply:
Done